env.sh

```diff
@@ -0,0 +1,11 @@
+export FLASK_APP="hushline/__init__.py"
+export REDIS_URI="memory://"
+export SECRET_KEY=$(openssl rand -hex 32)
+export ENCRYPTION_KEY=$(python3 -c 'from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())')
```

Comment:
We should just hardcode secrets here so that logins don't break every time the app restarts. If the GH action that checks for secrets yells at us, we shod make an exception.